src/gateway/environments/jsdom-sixteen/jsdom-sixteen-configuration.js

```diff
@@ -6,6 +6,7 @@ import type {IJSDOMSixteenConfiguration} from "./types.js";
  * environment.
  */
 export class JSDOMSixteenConfiguration implements IJSDOMSixteenConfiguration {
+    +registrationCallbackName: string;
```

Comment:
Rather than having a magic string, I thought it better if the webapp implementation of the `render-gateway` service could specify what the registration callback should be. That way we can rework that all inside the webapp repo.

Reply:
Oh nice, I like that!